src/auth/downscopedclient.ts

```diff
@@ -48,10 +53,18 @@ interface CredentialsWithResponse extends Credentials {
   res?: GaxiosResponse | null;
 }
 
+/**
+ * Internal interface for tracking and returning Downscoped access token
```

Comment:
"the downscoped access token's expiration time"

Reply:
this repo not always using 's, and used capped for Downscoped, I might have to follow the styles.